src/isIsogram.test.js

```diff
@@ -6,4 +6,28 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
```

Comment:
The task description mentions that the function should be case-insensitive. Consider adding test cases with mixed-case inputs to verify this behavior.